scripts/variantstore/wdl/GvsWithdrawSamples.wdl

```diff
@@ -102,6 +117,7 @@ task WithdrawSamples {
   }
   output {
     Int num_rows_updated = read_int("rows_updated.txt")
+    File new_samples_file = "new_samples.txt"
```

Comment:
For clarity, maybe change `num_rows_updated` to `num_samples_withdrawn` and `new_samples_file` samples_not_yet_ingested_file`?

Reply:
👍